scroll-animations-1/Overview.bs

```diff
@@ -145,9 +161,9 @@ spec: cssom-view-1; type: dfn;
 
 	Progress (the [=timeline/current time=]) for a [=scroll progress timeline=]
 	is calculated as:
-	<var>[=scroll offset=]</var> &div;
-	(<var>[=scrollable overflow=] size</var> &minus;
-		<var>[=scroll container=] size</var>)
+	<var ignore=''>[=scroll offset=]</var> &div;
+	(<var ignore=''>[=scrollable overflow=] size</var> &minus;
+		<var ignore=''>[=scroll container=] size</var>)
```

Comment:
Suppressing bikeshed warnings about a local variable only being used once.  The use of var makes the links italicized, but otherwise serves no purpose.